packages/app/src/specs/generators/FileChooser.vue

```diff
@@ -54,7 +54,7 @@
       <template #no-results>
         <NoResults
           empty-search
-          :search="noResults.search"
+          :search-term="noResults.search"
```

Comment:
Renamed this in passing since it's not always a `specFilterModel`. This component is used by several things (like file search, editor search). `searchTerm` seems a bit clearer to me than just search.